frame/composable-support/src/rpc_helpers.rs

```diff
@@ -1,79 +1,42 @@
-use codec::{Codec, Decode, Encode};
+use codec::{Decode, Encode, MaxEncodedLen};
 #[cfg(feature = "std")]
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
-
-/// https://github.com/interlay/interbtc/blob/a7c0e69ac041176a2531bafb1c4e35cbc2f7e192/crates/oracle/rpc/runtime-api/src/lib.rs#L10
```

Comment:
Original issue mentioned in this link https://github.com/paritytech/substrate/issues/4641 seems still relevant right? i.e the JSON serde has issues with `u128`

Reply:
Yeah so a number cannot be u128 in JSON thats the reason AFAIK

Reply:
Normally big numbers are represented as string in JSON, it's strange that this is not getting fixed